install/resources/upgrade49.php

```diff
@@ -31,6 +31,20 @@ function upgrade49_dbchanges()
 
 	$db->delete_query("settings", "name='allowyahoofield'");
 
+	if (!$db->field_exists("isunique", "sessions"))
+	{
+		switch ($db->type)
+		{
+			case "pgsql":
+			case "sqlite":
+				$db->add_column("sessions", "isunique", "smallint NOT NULL default '0'");
+				break;
+			default:
+				$db->add_column("sessions", "isunique", "tinyint(1) NOT NULL default '0'");
```

Comment:
Installer DB structure specifies a key for `isunique` for MySQL, but the upgrade script doesn't.